### PR TITLE
Data prep - Account for population change

### PR DESCRIPTION
**Pull request by Eleonore9**
Calculates the number of extra individuals who will be added to the population over the course of the projection due to population growth, and returns a dataset with a row for each of these individuals for each simulation.

### CODE REVIEW

File: src/witan/send/send.clj [hunk 1]
```diff
@@ -135,7 +159,61 @@
...
 
+(defn lag
+  "Dataset must be kept chronologically ordered in order to work correctly.
+   Shifts the specified column down by the specified number of rows, filling
+   in zeros at the top of the column and dropping the shifted values at the end."
+  ([dataset col-key]
+   (-> dataset
+       (ds/to-map)
+       (col-key)
+       (#(cons 0 %))
```
Comment (acron0) [inline src/witan/send/send.clj]: I can see why you've done this but it's ugly

File: src/witan/send/send.clj [hunk 2]
```diff
@@ -135,7 +159,61 @@
...
+       (butlast)))
+  ([dataset col-key lag-amount]
+   (-> dataset
+       (ds/to-map)
+       (col-key)
+       (#(concat (repeat lag-amount 0) %))
+       (as-> d (drop-last lag-amount d)))))
+
```
Comment (acron0) [inline src/witan/send/send.clj]: Both the lamba and `as->` are here to work around the thread-first so I suggest a two-step approach instead

File: src/witan/send/send.clj (hunk 1 above)
Comment (Eleonore9) [inline src/witan/send/send.clj]: @acron0 could you be more descriptive

Comment (acron0) [inline src/witan/send/send.clj]: Don't include `cons` in the thread-first

**Pull request merged**